package/upgrade/addons/monitoring_addon.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: harvesterhci.io/v1beta1
+kind: Addon
+metadata:
+  name: rancher-monitoring
+  namespace: cattle-monitoring-system
+spec:
+  repo: http://harvester-cluster-repo.cattle-system.svc/charts
+  version: "100.1.0+up19.0.3"
```

Comment:
note: the `version` will be further updated when applying addon, thus this PR can be tested without rely on https://github.com/harvester/harvester-installer/pull/492

when ISO is built without 492, the upgraded `rancher-monitoring` has version "100.1.0+up19.0.3"
with 492, the upgraded `rancher-monitoring` has version "102.0.0+up40.1.2"